modules/__tests__/keyframes-test.js

```diff
@@ -1,7 +1,8 @@
 var styleElement;
 var exenv;
 
-var {Component} = require('react');
+var React = require('react/addons');
```

Comment:
You can keep these as `var {Component} = require('react/addons');` any good reason to change it to two lines?